src/status_im/utils/contenthash.cljs

```diff
@@ -15,21 +13,12 @@
              (nil? ipld))
     (str "0xe301" (hex/encode (b58/decode hash)))))
 
-(defn decode [contenthash]
-  (when (and contenthash
-             (string/starts-with? contenthash "0xe3011220")
-             (= 74 (count contenthash)))
+(defn decode [hex]
+  (when (and hex (not= hex "0x")
+             (string/starts-with? hex "0xe30101701220")
```

Comment:
I doubt that all of the contenthash we are decoding have content type `dag-pb`

Reply:
That's the only one described in the spec. So far only one website follows the spec, `pac-txt.eth`

We can revisite later

(besides you implemented that part)

Reply:
no it's not, the spec is https://github.com/multiformats/multicodec/blob/master/table.csv
the EIP is just giving an exemple and it has typos

Reply:
I can accept the PR with this change 70 is wrong for sure

Reply:
@decanus could you clarify please?